src/System.Security.Cryptography.Csp/tests/Configurations.props

```diff
@@ -4,6 +4,8 @@
     <BuildConfigurations>
       netstandard-Windows_NT;
       netcoreapp-Windows_NT;
+      netstandard-Unix;
+      netcoreapp-Unix;
```

Comment:
@joperezr is there a preference of netcore or netstandard if the APIs\tests are the same? I assume both would be redundant